components/Card.js

```diff
@@ -1,5 +1,5 @@
-import Image from 'next/image'
-import Link from '@/components/Link'
+import Image from './Image'
+import Link from './Link'
```

Comment:
Is it better to import the `Link` component this way instead of `@/components/Link`?

Reply:
It's just my preference to import with the relative path if they are in the same folder